tests/snippets/ini/test_quoted_entries.txt

```diff
@@ -0,0 +1,36 @@
+---input---
+[section]
+    key 1 = "value1"
+    key 2 = "value2" # comment
+    key 3 = "value3 ; value 3bis" ; comment
+
+---tokens---
+'[section]'   Keyword
+'\n    '      Text.Whitespace
+'key 1'       Name.Attribute
+' '           Text.Whitespace
+'='           Operator
+' '           Text.Whitespace
+'"'           Literal.String
+'value1'      Literal.String
+'"'           Literal.String
+'\n    '      Text.Whitespace
+'key 2 = '    Literal.String
```

Comment:
this is actually highlighting a bug in the current implementation, right?

Reply:
Yes. Thanks for spotting this. Tracked in #2835